yt/frontends/ytdata/tests/test_unit.py

```diff
@@ -2,8 +2,17 @@
 import shutil
 import tempfile
 
+import numpy as np
+
 from yt.convenience import load
-from yt.testing import assert_fname, fake_random_ds, requires_file, requires_module
+from yt.frontends.stream.data_structures import load_uniform_grid
```

Comment:
this line will need fixing if the yt.loaders PRs go in first

Reply:
No worries, thanks. I'll be ready.